client.go

```diff
@@ -437,6 +445,27 @@ func (c *client) Range(ctx context.Context, r *RangeOp) ([]DomainObject, string,
 	return objectArray, token, nil
 }
 
```

Comment:
Add some docs here -- specifically around how to get the context into the function for go noobs